src/main/java/com/notably/model/suggestion/SuggestionItem.java

```diff
@@ -0,0 +1,19 @@
+package com.notably.model.suggestion;
+
+/**
+ * Represents the instance of the suggested item.
+ */
+interface SuggestionItem {
+    /**
+     * Gets the note path and/ or title displayed in the UI, e.g. "open -t root/cs2103".
+     * @return The note's path and/ or title.
```

Comment:
Since `getDisplayText()` sounds a lot more general than "path/title of the note", let's make this description more generic? Instead of describing how this method will return the path/title of the note, how about "Gets a user-friendly text suggestion to be displayed to the user"?